src/NCronJob/Configuration/DynamicJobFactory.cs

```diff
@@ -60,7 +60,7 @@ public Task RunAsync(IJobExecutionContext context, CancellationToken token)
         var arguments = new object[parameters.Length];
         for (var i = 0; i < parameters.Length; i++)
         {
-            if (parameters[i].ParameterType == typeof(JobExecutionContext))
```

Comment:
As `JobExecutionContext` isn't a public type, dynamic jobs can't even declare it as parameter.
Switching this to `IJobExecutionContext` aligns with the what the [doco](https://docs.ncronjob.dev/features/minimal-api/) advertizes